base/applications/cmdutils/replace/replace.c

```diff
@@ -437,15 +425,13 @@ INT cmd_replace (LPTSTR param)
                     break;
                 default:
                     invalid_switch(arg[i]);
-                    freep(arg);
-                    return 0;
+                    return 1; /* Error */
```

Comment:
Has this been checked on Windows? (checking the %ERRORLEVEL% set if you make the Windows' `replace` command fail)

Reply:
Fixed.